app/src/androidTest/assets/features/bluetoothConnection.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Bluetooth connection service
+  As a user
+  I want to be able to pair devices using Bluetooth to exchange data
+  So that I can exchange data with someone using my mobile device
+
+  Scenario Outline: Connect with a device from the paired list
```

Comment:
Don't you mean the device list? The next scenario outline (on line 11) is already about pairing with a device that you're already paired with